accepted/0000-isolated-workspaces.md

```diff
@@ -0,0 +1,177 @@
+- Start Date: 2018-02-12
+- RFC PR: (leave this empty)
+- Yarn Issue: (leave this empty)
+
+# Summary
+
+Running a workspace in isolation should be as easy as running all workspaces together. Running a yarn command in
```

Comment:
nit: "Running a workspace" is a bit misleading, word missed?

Reply:
Maybe it could say 

Working on a workspace in isolation should be as easy as working on all workspaces together.